### PR TITLE
Userland Blog Post

### PR DESCRIPTION
**Pull request by zeke**
A followup to the design work in https://github.com/electron/electron.atom.io/pull/617

This is ready for review. Would like to publish it Tuesday (tomorrow) around 10am Pacific.

cc @electron/maintainers

### CODE REVIEW

File: _posts/2016-12-20-userland.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+[![github-contributors](https://cloud.githubusercontent.com/assets/2289/21205352/a873f86c-c210-11e6-9a92-1ef37dfc986b.png)](https://electron.atom.io/userland)
+
+## Origins of Userland
+
+Userland is where people in software communities come together to share tools and ideas.
+The term originated in the Unix community, where it referred to
+any program that ran outside of the kernel, but today it means something more.
+When people in today's Javascript community refer to userland, they're usually
+talking about the <a href="http://npm.im">npm package registry</a>. This is where the majority of experimentation and
```
Comment (malept) [inline _posts/2016-12-20-userland.md]: HTML link...

**Comment by malept** [comment]
Also, all of the `https://electron.atom.io` links should drop the `s`.

**Comment by zeke** [comment]
Thanks, @malept. I fixed the markdown link and the https URLs 👍 

Wonder if there's an atom plugin that will fix the line lengths for me...

**Comment by zeke** [comment]
This is live at http://electron.atom.io/blog/2016/12/20/userland, but feedback is still welcome. 🙏

**Pull request merged**